packages/drizzle/src/postgres/schema/traverseFields.ts

```diff
@@ -936,30 +936,6 @@ export const traverseFields = ({
 
         break
 
-      case 'join': {
```

Comment:
Even if we are not using the relationship any more, I'm wondering if it isn't good to keep just for somebody who might want to do their own drizzle queries later.

Reply:
Since we have `afterSchemaInit` thing, they can add their relationships there as they like. I also feel like we should have some nice drizzle types generation, as for now it's not easy to use those.